src/components/DropZone/DropZone.scss

```diff
@@ -15,12 +15,17 @@ $dropzone-overlay-border-color-error: color('red');
 $dropzone-overlay-background: color('indigo', 'lighter');
 $dropzone-overlay-background-error: color('red', 'lighter');
 
+.DropZoneWrapper {
+  height: inherit;
+}
+
 .DropZone {
   position: relative;
   display: flex;
   justify-content: center;
   background-color: $dropzone-background;
   border-radius: $dropzone-border-radius;
+  height: inherit;
```

Comment:
This will allow for a fixed height above 300px on the `DropZone` container to be set by the consumer. I'm open for discussion if we should put a constraint on the maximum height the `DropZone` container should be.

Reply:
I don't there needs to be a maxheight